api/src/test/java/com/github/eyefloaters/console/api/support/EnabledOperationFilterTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.eyefloaters.console.api.support;
+
+import jakarta.ws.rs.core.Response.Status;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+
+import static com.github.eyefloaters.console.test.TestHelper.whenRequesting;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+
+@QuarkusTest
+@TestProfile(OperationFilterTestProfile.class)
+class EnabledOperationFilterTest {
+
+    @Test
+    void testDescribeTopicNotAllowed() throws Exception {
+        whenRequesting(req -> req.get("/api/kafkas/{clusterId}/topics/{topicId}", "kafka-1", "topic-1"))
+            .assertThat()
+            .statusCode(is(Status.METHOD_NOT_ALLOWED.getStatusCode()))
+            .body("errors.size()", is(1))
+            .body("errors.status", contains("405"))
+            .body("errors.code", contains("4051"));
```

Comment:
How are these error code assigned?

Reply:
The error `status` is simply the HTTP code and the `code` is the status + a sequential number as new ones are added. Just a basic system to keep them somewhat aligned and have a unique `code`. See `com.github.eyefloaters.console.api.support.ErrorCategory` and nested classes.

https://github.com/eyefloaters/console/blob/c580ce35ebb244097cc70dd20f0060abaac836fa/api/src/main/java/com/github/eyefloaters/console/api/support/ErrorCategory.java#L17

Reply:
OK. I was mainly asking to ensure there's a single place to refer to for all those codes and to ensure we don't duplicate codes. What you have these seems good enough for now.